mozanalysis/bootstrap.py

```diff
@@ -0,0 +1,34 @@
+import numpy as np
+from functools import partial
+
+
+def bootstrap(iteration, stat_fn, broadcast_data):
+    np.random.seed()
+    n = len(broadcast_data.value)
+    randints = np.random.randint(0, n, n)
+    return stat_fn(broadcast_data.value[randints])
+
+
+def run_bootstrap(sc, data, stat_fn, num_iterations=1000, confidence_level=0.95):
```

Comment:
Let's default to 2000 -- 1000 is actually a bit low

Reply:
Can we add docstrings to this? Doesn't have to be in this review, but we should generate docs before we call this done